noxfiles/dev_nox.py

```diff
@@ -100,7 +119,7 @@ def test_env(session: Session) -> None:
     session.notify("teardown", posargs=["volumes"])
 
     session.log("Building images...")
-    build(session, "dev")
+    build(session, fides_image)
     build(session, "admin_ui")
     build(session, "privacy_center")
```

Comment:
@NevilleS [mentioned](https://github.com/ethyca/fides/pull/2241#discussion_r1087962504) needing to add `test` to the list of apps that get additional images deployed in order to get privacy center built, but I think this line covers that? let me know if I'm missing something though

Reply:
this seems right to me? not sure what he might've been referring to 🤔